src/scss/components/_form.scss

```diff
@@ -59,18 +115,21 @@
   }
 
   &__label {
-    display: inline-block;
+    display: block;
     margin-bottom: 6px;
     color: get-color((color: blackNeutral, opacity: .9));
-    font-size: 15px;
+    font-size: 13px;
     font-weight: 700;
   }
 
   &__label-link {
     margin-left: 10px;
+    float: right;
```

Comment:
@eduardojmatos Nice solution with the text-align:right. I completely forgot about that option :)